src/renderer/features/contacts/ContactFilter/model/__tests__/contact-filter.test.tsx

```diff
@@ -0,0 +1,74 @@
+import { allSettled, fork } from 'effector';
+
+import { filterModel } from '../contact-filter';
+import { contactModel } from '@entities/contact';
+
+const contacts = [
+  {
+    id: 1,
+    name: 'name 1',
+    address: '222',
+    accountId: '0x333',
+    matrixId: '@444',
+  },
+  {
+    id: 5,
+    name: 'name 6',
+    address: '777',
+    accountId: '0x888',
+    matrixId: '@999',
+  },
+];
+
+describe('features/contacts/model/contact-filter-model', () => {
+  test('should return all contacts if no search query', async () => {
+    const scope = fork({
+      values: new Map().set(contactModel.$contacts, contacts),
+    });
+
+    expect(scope.getState(filterModel.$contactsFiltered)).toHaveLength(2);
```

Comment:
To not depend on specific value
```suggestion
    expect(scope.getState(filterModel.$contactsFiltered)).toHaveLength(contacts.length);
```